src/Symfony/Component/ErrorHandler/Exception/FlattenException.php

```diff
@@ -26,17 +26,40 @@
  */
 class FlattenException extends LegacyFlattenException
 {
+    /** @var string */
     private $message;
+
+    /** @var int|string */
     private $code;
+
+    /** @var self|null */
     private $previous;
+
+    /** @var array */
     private $trace;
+
+    /** @var string */
     private $traceAsString;
+
+    /** @var string */
     private $class;
+
+    /** @var int */
     private $statusCode;
+
+    /** @var string */
     private $statusText;
+
+    /** @var array */
     private $headers;
+
+    /** @var string|null */
```

Comment:
This is not consistent with the return type of the getter. So either this is wrong, or the return type is buggy.

Reply:
I think I was wrong here. I've updated the type.